README.md

```diff
@@ -148,6 +148,13 @@ You can mark samples as drafts if they are still work-in-progress. This means th
 }--->
 ```
 
+#### How to style examples
+
+You can’t reference external stylesheets when creating AMP pages. All styles must live in the head of the document using the tag `<style amp-custom>`.
+For AMPByExample, we are using templates, so that we can manage better the CSS, as suggested [here](https://www.ampproject.org/docs/guides/responsive/style_pages.html).
```

Comment:
We are using templates, in the sense that the css is added when the template is rendered, like what is happens here:https://github.com/ampproject/amp-by-example/blob/master/templates/example.html